### PR TITLE
image-layout: some schema

### PR DESCRIPTION
**Pull request by vbatts**
Signed-off-by: Vincent Batts <vbatts@hashbangbash.com>

### CODE REVIEW

**Comment by wking** [comment]
On Mon, Oct 31, 2016 at 12:48:28PM -0700, Vincent Batts wrote:
>     A schema/image-layout.json (15)

I like this name, but the pattern so far has been to use *-schema.json
for these entry-point files [1].  If we stick to that, this would be
image-layout-schema.json.

[1]: https://github.com/opencontainers/runtime-spec/pull/481#issuecomment-223274496

**Comment by jonboulle** [comment]
agree w/wking

File: schema/image-layout.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "description": "OpenContainer Image Layout Schema",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "id": "https://opencontainers.org/schema/image-layout",
+  "type": "object",
+  "properties": {
+    "imageLayoutVersion": {
+      "description": "version of the OCI image-layout",
+      "type": "string"
```
Comment (wking) [inline schema/image-layout.json]: This should probably be an `enum` with `1.0.0` as the only option.

**Comment by vbatts** [comment]
PTAL

**Comment by wking** [comment]
c7ff884 looks good to me.

**Comment by jonboulle** [comment]
is wanting rebase

**Comment by philips** [comment]
@vbatts needs rebase

**Comment by vbatts** [comment]
updated. PTAL.

**Comment by jonboulle** [comment]
there's no "MUST" here anymore

**Comment by vbatts** [comment]
@jonboulle you're right!

**Comment by vbatts** [comment]
updated. PTAL.

**Comment by jonboulle** [comment]
lgtm some

**Comment by wking** [comment]
75db5388 looks good to me.

**Comment by vbatts** [comment]
ugggggh. such rebase. Why does the schema fs have to include modtime?

updated.PTAL.

**Comment by vbatts** [comment]
.. omg ...
rebased. PTAL.

**Comment by jonboulle** [comment]
vincent

**Comment by jonboulle** [comment]
lgtm

**Comment by stevvooe** [comment]
LGTM

**Comment by jonboulle** [comment]
@vbatts lol

**Comment by vbatts** [comment]
:face_with_head_bandage: 
rebased. PTAL

**Comment by jonboulle** [comment]
lgtm

**Comment by vbatts** [comment]
bump

**Comment by stevvooe** [comment]
lgtm

**Pull request merged**